[docs] update ulimit in macOS Quick Start (#11939)
yugabyte/yugabyte-db: docs/content/latest/quick-start/install/macos.md

1. Each tablet maps to its own file, so if you experiment with a few hundred tables and a few tablets per table, you can soon end up creating a large number of files in the current shell. Make sure that this command shows a big enough value.

    ```sh
    $ launchctl limit
    ```

    We recommend the following soft and hard limits.

    ```output

>     maxproc     2500        2500

Reviewer: My machine has a maxproc value higher than this (2666). Are you sure you want me to lower it?

Author: Changed the check to have at least minimum values

    kern.maxfilesperproc=1048576
    ```

>     If this file does not exist, then for setting maxfiles soft and hard limits, create the file `/Library/LaunchDaemons/limit.maxfiles.plist` and insert the following:

Reviewer: ```suggestion
    If this file does not exist, create the following two files:

    \
    Create `/Library/LaunchDaemons/limit.maxfiles.plist` and insert the following:
```

      </plist>
    ```

>     For setting maxproc soft and hard limits, create the file `/Library/LaunchDaemons/limit.maxproc.plist` and insert the following:

Reviewer: ```suggestion
    \
    Create `/Library/LaunchDaemons/limit.maxproc.plist` and insert the following:
```

    ```xml
    <?xml version="1.0" encoding="UTF-8"?>
    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
      <plist version="1.0">
        <dict>
          <key>Label</key>
            <string>limit.maxproc</string>
          <key>ProgramArguments</key>
            <array>
              <string>launchctl</string>
              <string>limit</string>
              <string>maxproc</string>
              <string>2500</string>
              <string>2500</string>
            </array>
          <key>RunAtLoad</key>
            <true/>
          <key>ServiceIPC</key>
            <false/>
        </dict>
      </plist>
    ```

>     Ensure that the `plist` files are owned by `root:wheel` and has permissions `-rw-r--r--`. To take effect, you need to reboot your computer or run these command:

Reviewer: ```suggestion
    \
    Ensure that the `plist` files are owned by `root:wheel` and have permissions `-rw-r--r--`. To take effect, you need to reboot your computer or run the following commands:
```
